tests/pcmsolver/ccsd-pte/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+include(TestingMacros)
+
+add_regression_test(pcmsolver-ccsd-pte "psi;quicktests;pcmsolver;addon;ccsd;pte")
```

Comment:
Ah, ok.  Was just asking for my own edification, in case there's some CTest standards that I'm not aware of.  Approvals are in place, but I'll punt to one of y'all to figure out how you'd like to rebase and fix the test label.  +1 for the documentation with citations.